### PR TITLE
tighten permissions

### PR DESCRIPTION
**Pull request by simonfelding**
# Description

This tightens security a bit.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Small minor change not affecting the Ansible Role code (GitHub Actions Workflow, Documentation etc.)

## How Has This Been Tested?
it works fine

### CODE REVIEW

File: tasks/first_server.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     path: /etc/rancher/rke2
     owner: root
     group: root
-    mode: 0755
+    mode: 0750
```
Comment (MonolithProjects) [inline tasks/first_server.yml:9]: Modifying file and directory permissions to differ from those in the upstream RKE2 installation is not within the scope of this Ansible Role.

File: tasks/ingress-nginx.yml [hunk 2]
```diff
@@ -13,4 +13,4 @@
     dest: "{{ rke2_data_path }}/server/manifests/rke2-ingress-nginx-config.yaml"
     owner: root
     group: root
-    mode: 0664
+    mode: 0660
```
Comment (MonolithProjects) [inline tasks/ingress-nginx.yml:16]: Modifying file and directory permissions to differ from those in the upstream RKE2 installation is not within the scope of this Ansible Role.

File: tasks/remaining_nodes.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@
     path: /etc/rancher/rke2
     owner: root
     group: root
-    mode: 0755
+    mode: 0750
```
Comment (MonolithProjects) [inline tasks/remaining_nodes.yml:9]: Modifying file and directory permissions to differ from those in the upstream RKE2 installation is not within the scope of this Ansible Role.

File: tasks/standalone.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@
     path: /etc/rancher/rke2
     owner: root
     group: root
-    mode: 0755
+    mode: 0750
```
Comment (MonolithProjects) [inline tasks/standalone.yml:9]: Modifying file and directory permissions to differ from those in the upstream RKE2 installation is not within the scope of this Ansible Role.

**Review by MonolithProjects — CHANGES_REQUESTED** [review]

**Comment by simonfelding** [comment]
ok makes sense

**Pull request closed without merging**